app/src/main/res/layout/fragment_login.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <ImageView
+        android:id="@+id/vLoginHeaderImage"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/login_header_image_height"
+        android:layout_marginBottom="@dimen/login_header_image_margin_bottom"
+        android:background="@drawable/login_cover"
+        android:scaleType="fitXY"
+        android:src="@drawable/wolox_colors_gradient"
+        app:layout_constraintTop_toTopOf="parent" />
+
+
+    <ImageView
+        android:id="@+id/vLoginWoloxLogo"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/login_wolox_logo_image"
+        app:layout_constraintBottom_toTopOf="@+id/vLoginButton"
+        app:layout_constraintTop_toTopOf="@id/vLoginHeaderImage"
+        app:srcCompat="@drawable/wolox_logo_large" />
+
+    <EditText
+        android:id="@+id/vLoginFirstName"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/login_medium_height"
+        android:layout_marginLeft="@dimen/static_spacing_medium_extra"
+        android:layout_marginTop="@dimen/login_text_input_margin_top"
+        android:layout_marginRight="@dimen/static_spacing_medium_extra"
+        android:hint="@string/login_firstname"
+        android:inputType="text"
+        app:layout_constraintTop_toBottomOf="@id/vLoginHeaderImage" />
+
+    <EditText
+        android:id="@+id/vLoginLastName"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/login_medium_height"
+        android:layout_marginLeft="@dimen/static_spacing_medium_extra"
+        android:layout_marginRight="@dimen/static_spacing_medium_extra"
+        android:hint="@string/login_lastname"
+        android:inputType="text"
+        app:layout_constraintTop_toBottomOf="@id/vLoginFirstName" />
+
```

Comment:
The only thing to change is that, despite you copy the style guide, and its Ok, these input fields should be for email and password as the card say. So, you will have to change every reference to firstname and lastname, and that's all
![image](https://user-images.githubusercontent.com/43349271/89349477-9b6f2100-d684-11ea-9866-433b908c40ec.png)